paasta_tools/kubernetes/bin/paasta_secrets_sync.py

```diff
@@ -141,6 +145,74 @@ def sync_secrets(
     if not os.path.isdir(secret_dir):
         log.debug(f"No secrets dir for {service}")
         return True
```

Comment:
This function will return early if you have no paasta secrets defined, which is not desirable

Reply:
in fact, maybe you should create a separate `sync_boto_secrets` function and call it from `sync_all_secrets`